.github/workflows/api-deploy-production-ecs.yml

```diff
@@ -22,6 +22,7 @@ jobs:
                   fetch-depth: 0
 
             - name: Deploy API to Production
+              id: deploy-api
```

Comment:
Out of curiosity: why do we need this ID?

Reply:
It's used here: https://github.com/Flagsmith/flagsmith/pull/1433/files#diff-32d8dc41c7ce51704465d214b372550275a77836ff13b2ef332792086ab3151bR53